.github/actions/lint_code/action.yaml

```diff
@@ -12,12 +12,7 @@ runs:
       run: ./format.sh --check
     - if: ${{ failure() }}
       shell: bash
-      run:
-        'echo "Lint fails, please have a look on https://github.com/DataDog/system-tests/blob/main/docs/edit/format.md"
-
-        exit 1
-
-        '
+      run: 'echo "Lint fails, please have a look on https://github.com/DataDog/system-tests/blob/main/docs/edit/format.md" && exit 1'
```

Comment:
This is the only hand-rolled change since it confused the linter. Everything else is from `yamlfmt`